h2/src/main/org/h2/mvstore/tx/TransactionStore.java

```diff
@@ -437,12 +437,10 @@ void removeUndoLogRecord(int transactionId) {
     /**
      * Remove the given map.
      *
-     * @param <K> the key type
-     * @param <V> the value type
      * @param map the map
      */
-    <K, V> void removeMap(TransactionMap<K, V> map) {
-        store.removeMap(map.map, false);
+    void removeMap(TransactionMap map) {
```

Comment:
Please, add `<?, ?>`, the raw type creates a compiler warning.

Reply:
The same problem exists in `MVStore.removeMap()`. The type of `map` variable should be declared as `MVMap<?, ?>`.